.travis.yml

```diff
@@ -7,10 +7,10 @@ php:
 matrix:
   include:
     -
-        php: 7
+        php: 7.1
         env: dependencies=lowest
     -
-        php: 7
+        php: 7.1
         env: dependencies=highest
 
```

Comment:
minor; Let's add 7.2 beta already :)

Reply:
added

Reply:
And removed, it seems one of our dependencies isn't quite php 7.2 ready yet.

https://github.com/fgrosse/PHPASN1/pull/66